src/test/java/org/cryptomator/cryptofs/health/dirid/DirIdCheckTest.java

```diff
@@ -130,7 +143,11 @@ private static void initDirStructure(Path root, String line) throws UncheckedIOE
 				var file = line.substring(0, sep);
 				var contents = line.substring(sep + 3);
 				Files.createDirectories(root.resolve(file).getParent());
-				Files.writeString(root.resolve(file), contents, StandardCharsets.US_ASCII, StandardOpenOption.CREATE_NEW);
+				if (!contents.equals("[EMPTY]")) {
```

Comment:
Still: Please invert the if/else branches. Else should be the "default" case

Reply:
Fixed in 0df4948

Reply:
Now you're opening, closing, reopening, writing, closing... No more atomic checks. I don't understand what would be so wrong about a simple `if (contents.equals("[EMPTY]"))`. But since this is just test code, I give up.

Reply:
Whoopsie 😅 
![grafik](https://user-images.githubusercontent.com/9036915/126221787-431d4adf-4a4c-4865-9371-e3a4c5aece5d.png)

I read "revert" rather than "invert" and wondered what should be reverted... i'll fix this asap.